app/src/main/java/net/programmierecke/radiodroid2/PlayerService.java

```diff
@@ -661,7 +677,11 @@ private void sendMessage(String theTitle, String theMessage, String theTicker) {
                 .setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
                 .setSmallIcon(R.drawable.ic_play_arrow_white_24dp)
                 .setLargeIcon(radioIcon.getBitmap())
-                .addAction(R.drawable.ic_stop_white_24dp, getString(R.string.action_stop), pendingIntentStop);
+                .setStyle(new android.support.v4.media.app.NotificationCompat.MediaStyle()
+                        .setMediaSession(mediaSession.getSessionToken())
+                        .setShowActionsInCompactView(0, 2, 3 /* stop, play/pause, next */))
+                .addAction(R.drawable.ic_stop_white_24dp, getString(R.string.action_stop), pendingIntentStop)
```

Comment:
Vector drawables are great but only for API 21+. It crashes on older APIs due to them being in the notification.

Reply:
Ah, yes, `minSdkVersion=15`. I relied too much on android studio :) The recommended way to deal with this seems to just use [VectorDrawableCompat](https://developer.android.com/guide/topics/graphics/vector-drawable-resources#vector-drawables-backward-solution). What do you think?

Reply:
The issue is that notification is processed by OS itself and it doesn't know about any trick application does on its side. So PNGs.

Reply:
OK, I'll have to setup and Android 4.0 emulator anyway to see how things look like...

Reply:
Seems to be working fine on API 16, now.